packages/ariada-jsr/scan-evidence/scan-result-preview.html

```
@@ -0,0 +1,33 @@
+      th { background: #eef3f8; }
+    </style>
+  </head>
+  <body>
+    <main>
+      <section class="banner">
+        <h1>S72 JSR package publish — scan-result preview</h1>
+        <p>This preview is the visual surface captured for the report. It is a scan-result preview, not a report-only screenshot, because it shows the generated consumer command, dry-run result and verification table for the JSR channel package.</p>
+        <div class="status">Dry-run validation passed</div>
+        <h2>Delegated Ariada command</h2>
+        <pre>npx --yes @ariada-org/cli@0.1.0 scan https://example.test/jsr-consumer --output-dir ./ariada-output --format both --severity-threshold moderate --domains accessibility,security,privacy</pre>
+        <table><thead><tr><th>Evidence case</th><th>Command</th><th>Status</th></tr></thead><tbody><tr><td>TypeScript source check</td><td>node_modules/.bin/tsc -p packages/ariada-jsr/tsconfig.json --noEmit</td><td>PASS</td></tr><tr><td>Deno consumer fixture check</td><td>deno check packages/ariada-jsr/examples/consumer.ts</td><td>PASS</td></tr><tr><td>ESLint</td><td>node_modules/.bin/eslint packages/ariada-jsr/src packages/ariada-jsr/tests --max-warnings=0</td><td>PASS</td></tr><tr><td>Vitest</td><td>node <vitest.mjs> run packages/ariada-jsr/tests/mod.test.ts</td><td>PASS</td></tr><tr><td>JSR dry-run</td><td>pnpm --filter @ariada-org/ariada-jsr validate:jsr</td><td>PASS</td></tr></tbody></table>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)